test/Altinn.App.Core.Tests/Helpers/ShadowFieldsConverterTests.cs

```diff
@@ -45,6 +45,8 @@ public void ShouldRemoveShadowFields_WithPrefix()
         Assert.Contains("AltinnSF_", serializedDataWithoutModifier);
 
         var modifier = new IgnorePropertiesWithPrefix("AltinnSF_");
+
+        // TODO: could be cached in a dictionary with the prefix as key, since serializer options _should_ be static/reused
```

Comment:
No, we should not use a dictionary to cache the json serializer in tests, because they won't be reused anyway.

Reply:
Removed the comment since I opened a PR for this now: #591